scss/helpers/_colored-links.scss

```diff
@@ -1,12 +1,18 @@
+// stylelint-disable function-name-case
+
+// All-caps `RGBA()` function used because of this Sass bug: https://github.com/sass/node-sass/issues/2251
 @each $color, $value in ("primary": $accessible-orange, "light": $light) { // Boosted mod: restrict to primary and light
```

Comment:
**[Out of PR]** Should we add `dark` helper here ?

Reply:
Please create a dedicated issue if it needs to be tackled or at least studied